go.mod

```diff
@@ -43,7 +43,7 @@ require (
 	github.com/sirupsen/logrus v1.6.0
 	github.com/sonatard/noctx v0.0.1
 	github.com/sourcegraph/go-diff v0.5.3
-	github.com/spf13/cobra v1.0.0
+	github.com/spf13/cobra v1.0.1-0.20200710201246-675ae5f5a98c
```

Comment:
this is the from the master branch (after related PR is merged), we can wait for official release as well.

Reply:
I asked @spf13 about the release date, hopefully he will answer
https://gophers.slack.com/archives/CD3LP1199/p1594518169033000

Reply:
So if we wait the cobra release, then could you mark it as WIP?

Reply:
or we can merge

Reply:
Oh I am not sure what the practice we follow here in this project, so keen to hear inputs from other you have more experience working on this :). Marking this as draft will not pop up notification.

/cc @golangci/core-team

Reply:
you can add `WIP` prefix in title, then the WIP check will block this PR from merging

Reply:
cool hope that team will still check WIP PR, I personally don't :laughing: